modules/snooker/src/test/ts/module/ephox/snooker/test/Bridge.ts

```diff
@@ -51,7 +51,25 @@ const generators: Generators = {
   colgroup: () => SugarElement.fromTag('colgroup')
 };
 
+const createCell = (): SugarElement<HTMLTableDataCellElement> => {
```

Comment:
```suggestion
const createCell = (): SugarElement<HTMLTableCellElement> => {
```

This shouldn't be a `Data` element (see DOM Interface in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td).